webpack/stories/docs/LegacyJs.md

```diff
@@ -1,24 +1,39 @@
 # Legacy JS
 
 Foreman's legacy javascript is based on ruby on rails assets pipeline and located in `assets/javascripts`
+New features are most welcome on webpack, but you can still get data directly from redux's state and observe the state within the legacy js
 
+> ⚠️ **Warning**: Do not update react content with jquery or other DOM manipulation
+> ⚠️ **Warning**: Do not alter the state manually, only via actions
+> ⚠️ **Warning**: Avoid changing the actual DOM with react
 
-### Access webpack javascript
+![foreman-frontend](https://raw.githubusercontent.com/theforeman/foreman/396a0ca1760b2ae2340a39ea77ac9410831488f8/webpack/stories/docs/foreman-frontend-infra.png "Foreman Frontend Infrastrcutre")
```

Comment:
s/Infrastrcutre/Infrastructure/

Pointing to file on GH is because simply using a relative path to file does not work, right?

I would prefer not to rely on external sources, what do you think of [serving static files](https://storybook.js.org/docs/configurations/serving-static-files/) by storybook? Option 2 looks like it could work with markdown as well.